baked_in.go

```diff
@@ -1370,6 +1371,23 @@ func isURL(fl FieldLevel) bool {
 	panic(fmt.Sprintf("Bad field type %T", field.Interface()))
 }
 
+// isHttpURL is the validation function for validating if the current field's value is a valid HTTP(s) URL.
+func isHttpURL(fl FieldLevel) bool {
+	if !isURL(fl) {
+		return false
+	}
+
+	field := fl.Field()
+	switch field.Kind() {
+	case reflect.String:
+
+		s := strings.ToLower(field.String())
+		return strings.HasPrefix(s, "http://") || strings.HasPrefix(s, "https://")
```

Comment:
How about making use of `url`?
```code
   u, err := url.Parse(s)
   if err != nil || u.Scheme == "http" || u.Scheme == "https" {
      panic(err)
   }
```

Reply:
I agree, except for the panic portion, that this would be fat superior to actually validate it's a valid URL which is far more than just the leading scheme.